helpers/installcredprovider.sh

```diff
@@ -24,7 +24,14 @@ if [ -z ${USE_NET6_ARTIFACTS_CREDENTIAL_PROVIDER} ] || [ ${USE_NET6_ARTIFACTS_CR
   esac
 # Don't attempt to install .NET 8 without a set variable.
 elif [ ! -z ${USE_NET8_ARTIFACTS_CREDENTIAL_PROVIDER} ] && [ ${USE_NET8_ARTIFACTS_CREDENTIAL_PROVIDER} != "false" ]; then
-  FILE="Microsoft.Net8.NuGet.CredentialProvider.tar.gz"
+  # Self-contained versions are available in latest versions of the .NET 8 credprovider.
+  # To install a release with a specific runtime version set the `USE_RID_ARTIFACTS_CREDENTIAL_PROVIDER` enviornment variable.
```

Comment:
AZURE_ARTIFACTS_CREDENTIAL_PROVIDER_RID to match the version env var? Not sure the USE_ one should be copied